sounddevice_client.py

```diff
@@ -0,0 +1,52 @@
+"""
+Simple (hopefully) multiplatform client mimicing the linux command
+
+    arecord -f S16_LE -c1 -r 16000 -t raw -D default | nc localhost 43001
+
+which streams audio data from the microphone to the server.
+Tested on Mac Os.
+"""
+import sys
+import sounddevice as sd
+import socket
+import numpy as np
+from argparse import ArgumentParser
+
+
+parser = ArgumentParser(__doc__)
+parser.add_argument('--host', type=str, default='localhost', help='Host name')
+parser.add_argument('--port', type=int, default=43007, help='Port number')
+parser.add_argument('--chunk', type=int, default=1000, help='Chunk size in ms')
+args = parser.parse_args()
+
+
+
+# Audio configuration needed for whisper_online_server.py
+SAMPLE_RATE = 16000
+CHANNELS = 1
+CHUNK = 1024
```

Comment:
maybe rename it to

```suggestion
BLOCKSIZE = 1024
```

since its only used for this regards? =)